### PR TITLE
Add ch.swisstopo.geologie-geothermische_potenzialstudien_regional

### PR DESCRIPTION
**Pull request by daguer**

### CODE REVIEW

**Comment by ltclm** [comment]
to be merged too https://github.com/geoadmin/service-sphinxsearch/pull/376

**Comment by daguer** [comment]
@rebert : thanks to merge this PR. This layer go to prod next deploy (31.10).

File: chsdi/templates/htmlpopup/geothermische_potenzialstudien.mako [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+<%def name="table_body(c, lang)">
+<%
+    lang = lang if lang in ('fr','it','en') else 'de'
+    titel = 'titel_%s' % lang
+    weblink = 'weblink_%s' % lang
+    download = c['attributes']['download']
+%>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.land')}</td>       <td>${c['attributes']['land'] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.kanton')}</td>       <td>${c['attributes']['kanton'] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.nb_studien')}</td>       <td>${c['attributes']['nb_studien'] or '-'}</td></tr>
```
Comment (AFoletti) [inline chsdi/templates/htmlpopup/geothermische_potenzialstudien.mako]: This returns "-" when the value in the table is "0". (see UR for example)
Should display "0" instead

File: chsdi/templates/htmlpopup/geothermische_potenzialstudien.mako [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    jahr = c['attributes']['jahr'].split('##')
+    auftraggeber = c['attributes']['auftraggeber'].split('##')
+    weblink = c['attributes'][weblink].split('##')
+    download = c['attributes']['download'].split('##')
+%>
+
+<table class="table-with-border bohrungen_tiefer_500-extended">
+     <tr><th class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.land')}</th>       <td>${c['attributes']['land'] or '-'}</td></tr>
+     <tr><th class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.kanton')}</th>       <td>${c['attributes']['kanton'] or '-'}</td></tr>
+     <tr><th class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.nb_studien')}</th>       <td>${c['attributes']['nb_studien'] or '-'}</td></tr>
```
Comment (AFoletti) [inline chsdi/templates/htmlpopup/geothermische_potenzialstudien.mako]: This returns "-" when the value in the table is "0". (see UR for example)
Should display "0" instead

**Comment by AFoletti** [comment]
Pinging this one @daguer . It's needed for next deploy!

**Comment by daguer** [comment]
Hello @rebert 
Testlink: 
https://mf-chsdi3.int.bgdi.ch/ltgud_geol_potenzialstudien/shorten/7d607c9758 
and PR is squashed.
Thanks to merged this PR for next deploy (31.10)
Gruss

**Comment by rebert** [comment]
@AFoletti is this PR now ok for you?

**Comment by AFoletti** [comment]
Yes, everything looks good on my side

**Pull request merged**